proto/desmos/subspaces/v1beta1/subspace.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+package desmos.subspaces.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/desmos-labs/desmos/x/staging/subspaces/types";
+
+// Subspace contains all the data of a Desmos subspace
+message Subspace {
+  option (gogoproto.equal) = true;
+  option (gogoproto.goproto_stringer) = true;
+
+  string id = 1 [
+    (gogoproto.customname) = "ID",
+    (gogoproto.jsontag) = "subspace_id",
+    (gogoproto.moretags) = "yaml:\"subspace_id\""
+  ];
+
+  string name = 2 [ (gogoproto.moretags) = "yaml:\"name\"" ];
+
+  string owner = 3 [ (gogoproto.moretags) = "yaml:\"owner\""];
```

Comment:
Watching this, I think we should have another field named `creator` that allows to tell who was the creator of the subspace. We should also leave the `owner` as that might change over time while the creator stays the same. This would also serve the purpose to always identify a user that has created a subspace even when they do not specify an owner (if they want to make it immutable).